### ISSUE TITLE
System.TypeLoadException at WebhookRouter.Route method on .Net Core 3.x

### ISSUE THREAD

**Issue by Sagilio** [linvi/tweetinvi#971]
Hi there, I found It is a break change on `.Net Core 3.x`, the exception was be thrown here.
```csharp
public async Task<bool> Route(HttpContext context, IWebhookConfiguration configuration)
{
    ...
    context.Request.EnableRewind();
    var jsonBody = new StreamReader(context.Request.Body).ReadToEnd();
    ...
}
```
I update the project to 3.x, changed `context.Request.EnableRewind();` to `context.Request.EnableBuffering();`and `ReadToEnd()` to `ReadToEndAsync()`, then It works.

(More details about this can be found here: dotnet/aspnetcore#12505)

**Comment by linvi** [comment]
Thank you, I will be reviewing this as part of migration to version 5.0.

**Comment by linvi** [comment]
Fix was pushed in version 5.0.
Issue will be closed when tested with WebhookPlugin

**Comment by linvi** [comment]
Sort out the compilation warming of Tweetinvi.Webhooks in regards to using specific version of asp net dependencies.
